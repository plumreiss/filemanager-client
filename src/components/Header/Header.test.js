import { render } from "@testing-library/react";
import { Header } from ".";

describe("<Header />", () => {
  it("Should render correctly", () => {
    const component = render(<Header />);
    component.getByText("React Test App");
    expect(component.container).toMatchSnapshot();
  });
});
