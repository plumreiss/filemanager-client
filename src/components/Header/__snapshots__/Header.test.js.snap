// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render correctly 1`] = `
<div>
  <header
    class="container-fluid"
  >
    <div
      class="py-1 text-white row"
      style="background: rgb(255, 102, 102);"
    >
      <h1>
        React Test App
      </h1>
    </div>
  </header>
</div>
`;
